js/viz.js

```diff
@@ -59,33 +72,51 @@ const viz = {
   },
 
   updatePositions() {
-    this.allCircles
-      .attr('cx', (d) => d.x)
-      .attr('cy', (d) => d.y);
-    this.allLabels
+    this.circles
```

Comment:
we are not using SVGs, so this piece isn't needed too. However at this point I don't know how to update the coordinates. So I leave this as it is.